Improve documentation: Component Overview section (#9056)
langflow-ai/langflow: docs/docs/Concepts/concepts-components.mdx

Multiple components can be grouped into a single component for reuse. This is useful for organizing large flows by combining related components together, such as a RAG **Agent** component and an associated vector database component.

1. To select multiple components, use one of the following methods:  
   – Hold <kbd>Shift</kbd>, then click and drag to draw a selection box. Components must be entirely within the selection area to be selected.  

>    – Hold <kbd>Ctrl</kbd> (or <kbd>Cmd</kbd> on Mac), then click each component individually to select them one by one.  

Reviewer: ```suggestion
Alternatively, to select components for merging individually, hold <kbd>Cmd</kbd>, and then select the components.
```
